website/docs: Revamp Enterprise Docs. (#23920)
goauthentik/authentik: website/docs/enterprise/index.md

---
title: authentik Enterprise
description: "Licensed authentik features, license management, and commercial support"

> sidebar_label: Overview

Reviewer: ```suggestion
```

sidebar_position: 0
---

> authentik Enterprise adds licensed capabilities and commercial support to a self-hosted authentik installation. You install and upgrade authentik in the same way with or without an Enterprise license. Installing a license unlocks the Enterprise features that are already included in the release.

Reviewer: ```suggestion
authentik Enterprise adds licensed capabilities and commercial support to a self-hosted authentik installation. You install and upgrade authentik in the same way with or without an Enterprise license. Installing a license unlocks the Enterprise features.
```

## Choose an Enterprise plan

authentik offers two Enterprise plans:

- **Enterprise** adds Enterprise features and ticket-based support for qualifying subscriptions.
- **Enterprise Plus** adds customized support and service-level agreements, invoiced billing, volume pricing, multi-instance deployment support, and FIPS-compliant deployments for FedRAMP requirements.

> See [authentik pricing](https://goauthentik.io/pricing/) for current plan details and pricing. To discuss a trial, a custom agreement, or deployment requirements, contact [hello@goauthentik.io](mailto:hello@goauthentik.io).

Reviewer: ```suggestion
Enterprise licenses can be purchased via our [Customer Portal](https://customers.goauthentik.io/). Alternatively, if you'd like to organize a call with our team to discuss your needs, contact us via [hello@goauthentik.io](mailto:hello@goauthentik.io).
```

## Start using authentik Enterprise

> Use these pages based on what you need to do:

Reviewer: ```suggestion
Refer to our Enterprise documentation for further information:
```

- [Compare Enterprise features](./enterprise-features.md) and review [current plan pricing](https://goauthentik.io/pricing/).
- [Get started with authentik Enterprise](./get-started.md) on a new or existing authentik installation.
- [Manage Enterprise licensing](./manage-enterprise.md) to control organization access, purchase and manage licenses, and manage billing.
- [Enterprise support](./enterprise-support.md) to open and track support tickets.

> After you install a license, use the standard authentik documentation to configure the features that you need. Each applicable feature guide identifies Enterprise-only and preview functionality.

Reviewer: ```suggestion
After you install a license, use the standard authentik documentation to configure the features that you need.
```
